### ISSUE TITLE
Accessibility: menu toggle actions are not accessible with orca

### ISSUE THREAD

**Issue by deepak1556** [microsoft/vscode#91642]
Refs https://github.com/microsoft/vscode/issues/91063

OS Version: Ubuntu 18.04
Screenreader: Orca

Steps to Reproduce:

1. Navigate to a menu with toggle action, ex: View -> Show Minimap
2. Notice that orca reads the current state of the menuitem, whether checked or unchecked
2. Toggle the state, orca doesn't read the new state of the menuitem

**Comment by isidorn** [comment]
Thanks for filling this. This is the issue with the custom menu, that on chosing any menu item the item is not read out.
Compare it with the native menus, once you select the menu the screen reader will re-read its label with state. This does not happen for custom menus, thus forwarding to @sbatten

I do not believe this is a big issue, but I think we should be consistent with how native menus behave.

**Comment by sbatten** [comment]
custom menus are not default on linux so assigning to march

**Comment by isidorn** [comment]
Yeah definetely not urgent. March is great.
@deepak1556 were you using custom menus? IF you were using native menus, did you see the same behavior for example with native menus in Chrome?

**Comment by deepak1556** [comment]
I was using the native menus and just verified that the problem exists with chrome , firefox as well. Also the problem exists with custom menu, is it something we can fix for custom menus ?

**Comment by isidorn** [comment]
Yes, I believe we can fix for custom menus.
So keeping this issue open makes sense.

**Comment by sbatten** [comment]
After testing this on Windows and Linux, native menus also don't announce anything after selection. In all cases, the current state is announced before activating the menu entry.
